### PR TITLE
Added A Song

### PR DESCRIPTION
**Pull request by Ulvsblakk**
Play With Me (Variant 6) is easily my favorite song from the game. I wanted to add it.

### CODE REVIEW

**Comment by kaido1224** [comment]
Is that the Natsuki hunger song?

**Comment by sebastianN01** [comment]
Its when yuri death cg is on, Maybe add a dialogue from Monika about this variant? Perhaps something about Yuri?

**Comment by Ulvsblakk** [comment]
I did the thing.

File: Monika After Story/game/zz_music_selector.rpy [hunk 1]
```diff
@@ -26,6 +27,7 @@ init -1 python in songs:
         "<loop 17.451 to 119.999>mod_assets/bgm/ddlc_maintheme_80s.ogg"
     )
     FP_SAYO_NARA = "<loop 36.782>bgm/d.ogg"
+    FP_PLAYWITHME_VAR6 = "bgm/6s.ogg"
```
Comment (ThePotatoGuy) [inline Monika After Story/game/zz_music_selector.rpy]: Mmm I'll listen to this later, but I'm almost 100% certain that you're gonna need to make this loop seamlessly. Can you add a loop tag? The other songs have it, you should be able to infer how it works.

Comment (Ulvsblakk) [reply Monika After Story/game/zz_music_selector.rpy]: I'll work on it.

**Comment by Ulvsblakk** [comment]
It's in there

**Comment by ThePotatoGuy** [comment]
I changed the loop tag to be the same as the stock game one, as that works better.

**Comment by Ulvsblakk** [comment]
Oh dang. I didn't even know there was one lol

**Pull request merged**